src/components/WebexParticipantRoster/WebexParticipantRoster.js

```diff
@@ -15,7 +15,9 @@ import useParticipants from '../hooks/useParticipants';
 export default function WebexParticipantRoster({destination}) {
   const participants = useParticipants(destination);
 
-  const participantList = participants.map((participant) => <WebexParticipant personID={participant.personID} />);
+  const participantList = participants.map((participant) => (
+    <WebexParticipant personID={participant.personID} key={participant.personID} />
```

Comment:
Can you have the `key` attribute first, like we do in [activity stream](https://github.com/webex/components/blob/master/src/components/WebexActivityStream/WebexActivityStream.js#L41)?
```suggestion
    <WebexParticipant key={participant.personID} personID={participant.personID} />
```